config/prow/cluster/monitoring/mixins/lib/config.libsonnet

```diff
@@ -67,7 +67,7 @@ local config = {
   webhookMissingAlertInterval: '10m',
 
   // How many days prow hasn't been bumped.
-  prowImageStaleByDays: 7,
+  prowImageStaleByDays: {daysStale: 7, hoursToTrigger: '24h'},
```

Comment:
This is a breaking change for every other Prow instance if we just want to be able to update the alert files without requiring config updates. Thats probably ok for now since we haven't automated the update and can manually fix this for every other instance, but please avoid in the future.

nit: `prowImageStaleByDays` is now kind of incorrectly named. Please drop the `Days` suffix now that this is a struct. Similarly `hoursToTrigger` is a bit of a misnomer since the value contains units.

nit: The `hoursToTrigger` field would be a good field to add defaulting logic for. I think `24h` is suitable for most every instance.

Reply:
I have added defaults for both `daysStale` and `hoursToTrigger` at https://github.com/kubernetes/test-infra/pull/20256/files#diff-f0b2fe3f3f67579243edc6515cac8a84522abbe9633dcb4c62a786c52ac517bdR42, did you mean to default `hoursToTrigger` only?

Reply:
Defaulting both is great. `hoursToTrigger` just seemed a little more likely to be the same across multiple instances so I called that out.

Reply:
ack

Reply:
`hoursToTrigger` is still a misnomer and inconsistent with the `daysStale` field because it is actually a duration since it includes the `h` unit specifier.

Reply:
makes sense, updated to `eventDuration`